config/default.py

```diff
@@ -25,6 +25,11 @@
 SECURITY_MSG_PASSWORD_NOT_PROVIDED = ("Invalid username/password combination", "error")
 SECURITY_MSG_USER_DOES_NOT_EXIST = ("Invalid username/password combination", "error")
 
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.WARNING)
```

Comment:
The level is the minimum level for the logger to print (to stdout). By making it WARNING instead of DEBUG, we're saying don't write it out to the console unless it's worth knowing. You'll see that for the development configuration, we lower it back down to DEBUG so we can see what's going on. When we look at Heroku logs though, we don't need to see everything, just the relevant stuff.

You can take a look at the levels here: https://docs.python.org/2/library/logging.html#levels.

Speaking of which, instead of writing print statements when debugging, we should write logger.debug("WHAT IS THIS VALUE???").